skyportal/facility_apis/atlas.py

```diff
@@ -280,6 +273,7 @@ def get(request):
                         req.id,
                         instrument.id,
                         request.requester.id,
+                        session,
```

Comment:
No, this one is naughty. Can't steal a session for an executor!

Reply:
@stefanv makes sense. Removed.